src/css/app.sass

```diff
@@ -16,4 +16,8 @@
 
 .constrain
     max-width: 975px
+    margin: 0 auto
+
+.constrain-more
+    max-width: 600px
     margin: 0 auto
```

Comment:
do NOT use your arms, use your prettier syntax code formatter. I see difference in single PR with styles having 1 space and styles having 2 spaces. It is not tabulation error, use prettier everythere

![image](https://user-images.githubusercontent.com/59572810/220582128-d20922b5-947d-4c49-9329-15cb63dff4cb.png)